scripts/release_scripts/repo_specific_changes_fetcher_test.py

```diff
@@ -102,16 +106,22 @@ def mock_run_cmd(unused_cmd: str) -> str:
     def test_get_changes(self) -> None:
         def mock_get_changed_schema_version_constant_names(
             unused_release_tag_to_diff_against: str
-        ) -> List[str]:
-            return ['version_change']
+        ) -> List[str] | None:
```

Comment:
I think the canonical way to type this is `Optional[List[str]]` but I'm not sure. Ditto below

Reply:
Done